rtloader/test/rtloader/rtloader.go

```diff
@@ -165,11 +192,30 @@ func runFakeCheck() (string, error) {
 	runtime.LockOSThread()
 	state := C.ensure_gil(rtloader)
 
-	C.get_class(rtloader, C.CString("fake_check"), &module, &class)
-	C.get_attr_string(rtloader, module, C.CString("__version__"), &version)
-	C.get_check(rtloader, class, C.CString(""), C.CString("{\"fake_check\": \"/\"}"), C.CString("checkID"), C.CString("fake_check"), &check)
+	classStr := (*C.char)(helpers.TrackedCString("fake_check"))
+	defer C._free(unsafe.Pointer(classStr))
+	C.get_class(rtloader, classStr, &module, &class)
+
+	verStr := (*C.char)(helpers.TrackedCString("__version__"))
+	defer C._free(unsafe.Pointer(verStr))
+
+	C.get_attr_string(rtloader, module, verStr, &version)
+	defer C._free(unsafe.Pointer(version))
 
-	out, err := C.GoString(C.run_check(rtloader, check)), fetchError()
+	emptyStr := (*C.char)(helpers.TrackedCString(""))
```

Comment:
Note: If you want to remove most of `defer C._free(unsafe.Pointer(XXX))` you can create a local function TrackedAndRegisterCString that call `helpers.TrackedCString` and save the pointer into a slice and call a  "release" function at the end.